### DISCUSSION TITLE
Squert replacement Geo-IP mapping capability???

### DISCUSSION THREAD

**Discussion by adventurousmarc** [Security-Onion-Solutions/securityonion#1648] [Unsupported Versions]
I really enjoy the geo-IP mapping capability with its visual capability. Squert provided a good view in to that. So what replaced it. How can I get this function back? I would like a good method to view all the source IP's countries on a dashboard without having to dig-in to it. Thakns

**Comment by weslambert** [comment]
We don't have anything at the moment, but there's nothing stopping you from creating a custom Kibana dashboard based off of the `geo` values.

**Reply by eatinsundip** [reply]
Is this still not something security onion is capable of I assume? Would be a cool feature! :)

**Comment by dougburks** [comment]
It may not be a world map, but we do have pre-defined queries in Hunt to show you Source and Destination countries:
![image](https://user-images.githubusercontent.com/1659467/97193843-d6fb2200-177f-11eb-8809-ac0dd690c8e3.png)

**Comment by adventurousmarc** [comment]
Nope unfortunately not
